src/Security/src/Authorization.Certificate/CertificateAuthorizationBuilderExtensions.cs

```diff
@@ -66,6 +68,14 @@ public static AuthorizationBuilder AddOrgAndSpacePolicies(this AuthorizationBuil
             }
         });
 
+        if (IPAddress.TryParse(Environment.GetEnvironmentVariable("CF_INSTANCE_INTERNAL_IP"), out IPAddress? address))
```

Comment:
We _could_ reuse `CloudFoundryEnvironmentSettingsReader.InstanceIP` here, which makes tracking the usage of this environment variable a bit easier. However, that requires taking a dependency on the `Configuration.CloudFoundry` package, which may not be desired.